### DISCUSSION TITLE
请问用户名和密码是神马呀？

### DISCUSSION THREAD

**Discussion by Vivitoto** [jeessy2/ddns-go#699] [General]
请问docker部署的容器在浏览器打开后，要求登录，试了admin@example.com 和 changeme 发现不对

**Comment by WaterLemons2k** [comment]
用户名和密码位于 `配置文件` 中（默认位于当前用户目录下的 `.ddns_go_config.yaml` 文件中)

参考：https://github.com/jeessy2/ddns-go/wiki/FAQ#%E5%BF%98%E8%AE%B0%E7%94%A8%E6%88%B7%E5%90%8D%E6%88%96%E5%AF%86%E7%A0%81
